fitbenchmarking/fitbenchmarking
# Create Github action for Nightly install

#### Description of Work
This PR adds a job in the github actions to test that fitbenchmarking can be installed as expected using 
```
python -m pip install fitbenchmarking[bumps,DFO,minuit,SAS,numdifftools]
```

It then fixes any problems arising from this new test.

Fixes #753


#### Testing Instructions

1. Make sure that the new job in the github actions passes

Function: Does the change do what it's supposed to?

Tests: Does it pass? Is there adequate coverage for new code?

Style: Is the coding style consistent? Is anything overly confusing?

Documentation: Is there a suitable change to documentation for this change?
